HW11_1.py

```diff
@@ -0,0 +1,16 @@
+from datetime import datetime
+
+def my_deco (func):
+    def wrap():
+        func()
```

Comment:
Add `return` statement here to make in works with functions that return some value. And move your print before this statement

Reply:
It looks like it is not, the return is still missing